x-pack/plugins/uptime/server/lib/adapters/framework/adapter_types.ts

```diff
@@ -31,6 +31,8 @@ export interface UptimeCoreSetup {
 
 export interface UptimeCorePlugins {
   features: PluginSetupContract;
+  alerting: any;
+  elasticsearch: any;
```

Comment:
no types available?

Reply:
++, I agree this is evil. My reasoning for it was that when NP migration is finished this will be cleaned up, but that's a pretty poor excuse, especially because we likely won't have that done until 7.8. I'll see what I can do here.